src/copy/pvCopy.cpp

```diff
@@ -433,10 +432,19 @@ bool PVCopy::init(epics::pvData::PVStructurePtr const &pvRequest)
     PVStructurePtr pvMasterStructure = pvMaster;
     size_t len = pvRequest->getPVFields().size();
     bool entireMaster = false;
-    if(len==0) entireMaster = true;
     PVStructurePtr pvOptions;
-    if(len==1) {
-        pvOptions = pvRequest->getSubField<PVStructure>("_options");
+    if(len==0) {
+        entireMaster = true;
+    }
+    else {
+        // If "_" is in the request, but not in the master structure,
+        // then assume the top level PV structure is requested
+        PVStructurePtr masterFieldPtr = pvMaster->getSubField<PVStructure>("_");
+        PVStructurePtr requestFieldPtr = pvRequest->getSubField<PVStructure>("_");
```

Comment:
As written, there is no way to apply a filter to a structure that has a field named `_` — I'm not sure if we actually want that or not, but I did see some wording somewhere which suggested that field names should never start with an `_` character so I think this is reasonable.

Reply:
This is correct. If you think it this would be better, I would be okay changing this behavior and simply assuming that "_" always refers to the top level structure regardless of whether the structure contains field "_" or not, however unlikely this is.